src/views/storage/ResizeVolume.vue

```diff
@@ -24,7 +24,7 @@
             initialValue: selectedDiskOfferingId,
             rules: [{ required: true, message: 'Please select an option' }]}]"
           :loading="loading"
-          :placeholder="$t('label.offering.type')"
+          :placeholder="$t('label.diskoffering.type')"
```

Comment:
See the param is `diskofferingid`, so instead of this we can simply call it `label.diskoffering`

Reply:
Done